src/autoscaler/eventgenerator/config/config_test.go

```diff
@@ -33,6 +33,7 @@ var _ = Describe("Config", func() {
 logging:
   level: info
 http_client_timeout: 10s
+use_log_cache: false
```

Comment:
this is not needed as  the default is false.

Reply:
it is use to test the correct yaml mapping in [src/autoscaler/eventgenerator/config/config.go](https://github.com/cloudfoundry/app-autoscaler-release/pull/620/files#diff-f31936d1587116d960d4012ad492c43c7943dda45ad179e9f1c7aaf23ea1c66a)